Source/SVDFunctions/arm_svdf_state_s16_s8.c

```diff
@@ -21,8 +21,8 @@
  * Title:        arm_svdf_s8.c
  * Description:  S8 basic SVDF layer function with s16 state tensor
  *
- * $Date:        26 October 2022
- * $Revision:    V.2.0.2
+ * $Date:        8 November 2022
+ * $Revision:    V.2.1.0
```

Comment:
Please update the major revision here as it is a non-backwards compatible change..The same in other places if applicable.

Reply:
Done